Makefile

```diff
@@ -1052,6 +1052,7 @@ provision-install: python-install
 	$(INSTALL) -m 0755 setup/openchange_migrate $(DESTDIR)$(sbindir)/
 	$(INSTALL) -m 0755 setup/openchange_newuser $(DESTDIR)$(sbindir)/
 	$(INSTALL) -m 0755 setup/openchange_neworganization $(DESTDIR)$(sbindir)/
+	$(INSTALL) -m 0755 setup/openchange_group $(DESTDIR)$(sbindir)/
```

Comment:
What is the procedure for this?. Should I create a branch or ask Julio when he does the merge?